src/extension.ts

```diff
@@ -103,7 +107,7 @@ export async function activate(context: vscode.ExtensionContext) {
             callback: async () => await ext.awsContextCommands.onCommandLogin(),
             telemetryName: {
                 namespace: TelemetryNamespace.Aws,
-                name: 'credentialslogin'
+                name: 'credentialsLogin'
```

Comment:
@bryceitoc9 can you confirm the proper name of this metric?

Reply:
Confirmed all lowercase is correct.